src/include/duckdb/main/secret/secret.hpp

```diff
@@ -90,6 +90,11 @@ class BaseSecret {
 	      serializable(other.serializable) {
 		D_ASSERT(!type.empty());
 	}
+	BaseSecret(const BaseSecret &other)
```

Comment:
maybe instead of adding a const copy constructor, maybe just change the param of the other copy constructor to const? I don't think we need both?

Reply:
Absolutely! Made this change. Thanks for the review.